ui/lib/core/addon/templates/components/info-table-row.hbs

```diff
@@ -42,8 +42,6 @@
           @viewAll={{viewAll}}
           @wildcardLabel={{wildcardLabel}}
         />
-      {{else if isLink}}
-        <LinkTo @route="vault.cluster.secrets.backend.show" @model={{value.[0]}} >{{value}}</LinkTo>
       {{else}}
```

Comment:
This is not used, instead because the value comes in as an array, it's handled by the `infoTableItemArray` .  (e.g. for connections and roles in the show view).